### PR TITLE
[14.0][MIG] project_department

### PR DESCRIPTION
**Pull request by polqubiq**
migration to 11.0 to 14.0 of the module project_department

### CODE REVIEW

**Review by gmvictoria — APPROVED** [review]
Funtional Review ok. LGTM!

**Review by ajaniszewska-dev — APPROVED** [review]

File: project_department/__manifest__.py [hunk 1]
```diff
@@ -2,8 +2,8 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 {
     "name": "Project Department Categorization",
-    "version": "11.0.1.0.0",
-    "author": "Camptocamp, Daniel Reis, Odoo Community Association (OCA)",
+    "version": "14.0.1.0.0",
+    "author": "QubiQ, Camptocamp, Daniel Reis, Odoo Community Association (OCA)",
```
Comment (dreispt) [inline project_department/__manifest__.py:6]: Porting a module is not enough to grant authorship.
Please add yourself to the contributors list instead.

File: project_department/views/project.xml [hunk 2]
```diff
@@ -12,21 +12,6 @@
...
-         <field name="arch" type="xml">
-               <field name="partner_id" position="before">
-                  <field name="department_id"/>
-               </field>
-               <xpath expr="//search" position="inside">
-                  <filter string="Department" context="{'group_by':'department_id'}"/>
-               </xpath>
-         </field>
-     </record>
-
```
Comment (dreispt) [inline project_department/views/project.xml]: Removing a feature?

**Review by dreispt — CHANGES_REQUESTED** [review]
Can you follow the migration instructions, and have separate commits for Lint/Black, and migrations changes?
Also, please add the readme subdirectory with readme fragments.

**Pull request closed without merging**